w!

```diff
@@ -0,0 +1,430 @@
+# coding=utf-8
```

Comment:
extraneous file?

Reply:
sorry - have to be a bit careful with my vim command!